typesafety/test_str_format.yml

```diff
@@ -0,0 +1,33 @@
+---
+- case: ok__dot_access
+  parametrized:
+    - level: info
+    - level: debug
+    - level: warning
+    - level: error
+    - level: trace
+    - level: exception
+  main: |
+    from loguru import logger
+
+    class Foo:
+      bar = "baz"
+
+    foo = Foo()
+
+    logger.{{ level }}('The bar is "{0.bar}"', foo)
```

Comment:
@Delgan that is one case I'd not dream of :D thx @ThibaultLemaire  for spotting that.